jenkins/css-pipeline

```diff
@@ -9,7 +9,7 @@ pipeline {
    */
   environment {
     MAVEN_OPTS="-Xmx2048m -Xms1024M -Xss128M -XX:-UseGCOverheadLimit"
-    MVNOPT="-B -P ess-css-settings,platform-default,csstudio-composite-repo-enable,eclipse-sites"
+    MVNOPT="-B -P ess-css-settings,platform-default,csstudio-composite-repo-enable,eclipse-sites -Dmaven.repo.local=/home/jenkins/workspace/css-maven-repo/.m2 -Dcsstudio.composite.repo=/home/jenkins/workspace/${env.JOB_NAME}/ess-css-extra/ess_css_comp_repo"
```

Comment:
You can't assume /home/jenkins/workspace/${env.JOB_NAME}/
The jenkins user could be srv_icsjenkins. I think there is an env variable for the workspace.

But I guess Claudio also wants to be able to override that path.

Reply:
Completely agree and Claudio and I actually discussed this specific path, and that it should be relative rather than absolute. I did not change it in this pull request though, since I wanted to test the refactoring of the settings file without making further changes to the Jenkins pipeline.

Unless it's clear, I did not add this path, I merely moved it to a command line property from a settings file. You can see it in the deleted settings-for-jenkins.xml and the deleted sed-command on line 76 in this file.